src/runtime_src/xdp/profile/plugin/user/user_cb.h

```diff
@@ -22,22 +22,12 @@
 
 // These are the functions that are visible when the plugin is dynamically
 //  linked in.  XRT should call them directly
-extern "C"
-XDP_EXPORT
-void user_event_start_cb(unsigned int functionID,
-                         const char* label,
-                         const char* tooltip) ;
+extern "C" XDP_EXPORT void user_event_start_cb(unsigned int functionID, const char* label, const char* tooltip);
 
-extern "C"
-XDP_EXPORT
-void user_event_end_cb(unsigned int functionID) ;
+extern "C" XDP_EXPORT void user_event_end_cb(unsigned int functionID);
```

Comment:
Think It's preferable and to put exports and externs on separate lines.

Reply:
Yes, XRT code also follows that.